Mono.Debugging/Mono.Debugging.csproj

```diff
@@ -16,14 +16,10 @@
 
   <Import Project="..\Mono.Debugging.settings" />
 
-  <ItemGroup>
-    <ProjectReference Include="..\..\nrefactory\ICSharpCode.NRefactory\ICSharpCode.NRefactory.csproj" />
-    <ProjectReference Include="..\..\nrefactory\ICSharpCode.NRefactory.CSharp\ICSharpCode.NRefactory.CSharp.csproj" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="System.Buffers" Version="4.5.0" />
     <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.8.0" />
```

Comment:
May as well bump some of these while you're here. Immutable can go to 6.0.0. Roslyn can go to 4.4.0-3.22505.14.

Also you'll want to use msbuild properties for these versions so they can be overridden by the vsmac build. Just like here:

https://github.com/mono/debugger-libs/blob/a832ea7e752c85e0409eccfdc1a2650261be36df/Mono.Debugging.Soft/Mono.Debugging.Soft.csproj#L21-L28

You can use `$(NuGetVersionSystemCollectionsImmutable)` and `$(NuGetVersionRoslyn)`.

Reply:
Done